compiler/src/typed/includemod.re

```diff
@@ -131,12 +134,19 @@ let type_declarations =
 
 /* Inclusion between extension constructors */
 
-/*let extension_constructors ~loc env ~mark cxt subst id ext1 ext2 =
-  let mark = mark_positive mark in
-  let ext2 = Subst.extension_constructor subst ext2 in
-  if Includecore.extension_constructors ~loc env ~mark id ext1 ext2
-  then ()
-  else raise(Error[cxt, env, Extension_constructors(id, ext1, ext2)])*/
+let extension_constructors = (~loc, env, ~mark, cxt, subst, id, ext1, ext2) => {
+  let mark = mark_positive(mark);
+  // let ext2 = Subst.extension_constructor(subst, ext2);
```

Comment:
@belph I spent a few hours looking into why this substitution doesn't work and continuously came up blank. It only matters because we do [this check](https://github.com/grain-lang/grain/blob/5b0739ccbfce28bcfcf18f09c26c4542f4801046/compiler/src/typed/typemod.re#L895) after typechecking. I'm not really sure what it's for, as Grain doesn't allow separate interface files so the inferred signature should always match the actual signature. Unless it's some sort of sanity check? But then, the error messages are pretty cryptic.

Regardless, this doesn't work and I'm not really sure if it needs to.

Reply:
Hmm, okay. I think it at least merits a TODO, if not an investigation issue, though. One of these days, we really need to properly fix up the typing module.